custom/rch/utils.py

```diff
@@ -0,0 +1,83 @@
+from zeep.client import Client
+from requests.exceptions import HTTPError
+
+from .emails import (
+    notify_error_in_service,
+    notify_insecure_access_response,
+    notify_parsing_error,
+)
+
+from custom.rch.const import (
+    PROJECTID,
+    ID,
+    PASSWORD,
+    RCH_WSDL_URL,
+    VALID_AADHAR_NUM_LENGTH,
+    ICDS_CAS_DOMAIN,
+)
+from custom.rch.exceptions import MultipleMatchException
+
+from corehq.apps.es.case_search import CaseSearchES
+
+
+def etree_to_dict(t):
+    return {t.tag: map(etree_to_dict, t.iterchildren()) or t.text}
+
+
+def send_request_for_beneficiaries(for_date, state_id, beneficiary_type, district_id):
+    service_obj = _get_service()
+    if not PROJECTID or not ID or not PASSWORD:
+        raise NotImplementedError("You must set credentials for RCH API request")
+    # need to pass the start and end date but since it would be the same we pass 'for_date' twice
+    return service_obj(PROJECTID, ID, PASSWORD, for_date, for_date, state_id, beneficiary_type, district_id)
+
+
+def _initiate_client():
+    return Client(RCH_WSDL_URL)
+
+
+def _get_service():
+    return _initiate_client().service.DS_Data
+
+
+def _fetch_beneficiaries(for_date, state_id, beneficiary_type, district_id):
```

Comment:
some docstrings on these functions would be helpful to know what data they return